spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -619,14 +622,25 @@ def main(argv: Sequence[str]):
                 register_landmarks(ftmp_label, ftmp_template_label, paramregmulti.steps['0'].dof,
                                    fname_affine='straight2templateAffine.txt', verbose=verbose)
             except RuntimeError:
-                raise('Input labels do not seem to be at the right place. Please check the position of the labels. '
-                      'See documentation for more details: https://spinalcordtoolbox.com/user_section/tutorials/registration-to-template/vertebral-labeling.html')
+                printv(
+                    "Input labels do not seem to be at the right place. "
+                    "Please check the position of the labels. "
+                    "See documentation for more details: "
+                    "https://spinalcordtoolbox.com/user_section/tutorials/registration-to-template/vertebral-labeling.html",
+                    1, 'error')
```

Comment:
This one is also a change of behaviour, but even more clearly a bug.

Reply:
Nitpick: Is there value to instead re-raising a `RuntimeError` with this message, instead of using `printv`?

```python
raise RuntimeError('Input labels do not seem to be at the right place. Please check the position of the labels. '
                   'See documentation for more details: '
                   'https://spinalcordtoolbox.com/user_section/tutorials/registration-to-template/vertebral-labeling.html')
```

This sort of gets into the weeds w.r.t. raising exceptions vs. using `printv`, though...

I know our rough guideline is "printv for CLI functions, exceptions for API functions". But, in this scenario, the `main` function of `sct_register_to_template` is quite lengthy, with the kind of logic that potentially _should_ be an API function. So, perhaps the traceback would be useful for identifying the specific location in-code where things went wrong?

(This is maybe bikeshedding a bit, though, especially given the purpose of this PR, so feel free to dismiss this comment without giving it much thought if it would become a big discussion on exceptions vs. printv.)

Reply:
I wanted to just limit myself to fixing the lint, because otherwise I would probably have started changing lots of code more deeply! I would say the options here are:
* We want to give a user-facing message, in which case the right way (currently) is to use `printv(..., 'error')`. I think ths is reasonable, because we're in a CLI script.
* We want this code to generate an actual `RuntimeError`, in which case let's just not catch the existing `RuntimeError` in the first place? I'm fine with attaching a meaningful description to exceptions, but I consider those to be programmer-facing, not user-facing, so I'm not sure it would be warranted here.

Reply:
Sounds good!